chapters/common_pitfalls.md

```diff
@@ -0,0 +1,44 @@
+# Common Pitfalls for new Vulkan Developers
+
+This is a short list of commom assumptions and traps developers new to Vulkan can make.  
+
+### Validation Layers
+
+During developement, try to ensure that the Validation Layers are enabled. They are an invaluable tool for catching mistakes while using the Vulkan API. Parameter checking, object lifetimes, and threading violations all are part of the provided error checks. A way to reassure that they are enabled is to verify if the text "Debug Messenger Added" is in the output stream. More info can be found in the [Vulkan SDK](https://vulkan.lunarg.com/doc/sdk) documentation. It is first on the list because it is the first line of defense and should help reduce the time spent discovering where the mistake was made.
```

Comment:
"try to ensure" -> "ensure". I'd also include a link to a tutorial on how to enable the validation layers programmatically